ethers.nimble

```diff
@@ -9,7 +9,7 @@ requires "chronos >= 4.0.0 & < 4.1.0"
 requires "contractabi >= 0.6.0 & < 0.7.0"
 requires "questionable >= 0.10.2 & < 0.11.0"
 requires "json_rpc >= 0.4.0 & < 0.5.0"
-requires "serde >= 0.1.1 & < 0.2.0"
+requires "serde#e889b76dda0636421484dcda1535aa0911345244"
```

Comment:
I don't want ethers to be pinned to a specific git hash of serde. Can't you make a release of serde instead?

Reply:
Yes, good catch. This was meant to be changed when the serde PR was merged and a release cut. Updated.